patch_rebaser/patch_rebaser.py

```diff
@@ -426,6 +427,7 @@ def main():
 
     # Default values for options in patch_rebaser.ini
     defaults = {
+        'driver': 'downstream_driver',
```

Comment:
If this is an addition to patch_rebaser.ini, could you also update [patch_rebaser.ini.example](https://github.com/release-depot/patch_rebaser/blob/master/patch_rebaser/patch_rebaser.ini.example) with the new configuration option + default, and a short explanation?